docs/en/developer/udf_develop_guide.md

```diff
@@ -44,17 +45,28 @@ The SQL types corresponding to C++ types are shown as follows:
 ```c++
     struct UDFContext {
         ByteMemoryPool* pool;  // Used for memory allocation.
-        void* ptr;             // Used for the storage of temporary variables. Single-line functions don't use it at present.
+        void* ptr;             // Used for the storage of temporary variables for aggregrate functions.
```

Comment:
`void*` is not type safe ...

Reply:
please left you advice

Reply:
No idea... But user has more rate to write a unsafe code and boom themself. :(